src/main/java/com/thecookiezen/bussiness/containers/boundary/ContainerFetcher.java

```diff
@@ -1,14 +1,18 @@
 package com.thecookiezen.bussiness.containers.boundary;
 
+import com.github.dockerjava.api.DockerClient;
 import com.github.dockerjava.api.command.StatsCmd;
 import com.github.dockerjava.api.model.Container;
 import com.github.dockerjava.api.model.Info;
 
 import java.util.Collection;
 
 public interface ContainerFetcher {
-    Collection<Container> list();
-    Info getInfo();
+    Collection<Container> list(DockerClient dockerClient);
```

Comment:
\[Checkstyle\] WARNING: 'method def modifier' have incorrect indentation level 4, expected level should be 2\.